pkg/collector/python/check.go

```diff
@@ -385,7 +385,7 @@ func (c *PythonCheck) GetDiagnoses() ([]diagnosis.Diagnosis, error) {
 		}
 
 		if len(dw.RawError) > 0 {
-			d.RawError = errors.New(dw.RawError)
+			d.RawError = dw.RawError
```

Comment:
Would it make sense to remove `diagnosisJSONSerWrap` type? IIRC the only reason for it was inability to deserialize error types from json.

Reply:
@vickenty your recollection is perfect. I had some twinge in my head touching this line of code but the point you have made did not click. Thank you for reminding (changed is pushed) 🙏